app/components/post_list/post/body/index.tsx

```diff
@@ -12,17 +12,20 @@ import {THREAD} from '@constants/screens';
 import {isEdited as postEdited, isPostFailed} from '@utils/post';
 import {makeStyleSheetFromTheme} from '@utils/theme';
 
+import Acknowledgements from './acknowledgements';
 import AddMembers from './add_members';
 import Content from './content';
 import Failed from './failed';
 import Message from './message';
 import Reactions from './reactions';
 
 import type PostModel from '@typings/database/models/servers/post';
+import type UserProfile from '@typings/database/models/servers/user';
 import type {SearchPattern} from '@typings/global/markdown';
 
 type BodyProps = {
     appsEnabled: boolean;
+    currentUser: UserProfile;
```

Comment:
do we really need the entire UserProfile? more importantly this seems to be needed by the `Acknowledgements` which is not always mounted, so I guess `Acknowledgements` should be connected and get the full UserProfile or the props of the user needed in the component.